support-frontend/assets/helpers/tracking/acquisitions.ts

```diff
@@ -269,6 +269,37 @@ function getReferrerAcquisitionDataFromSessionStorage():
 		: null;
 }
 
+function getAcquisitionDataFromUtmParams():
+	| Record<string, unknown>
```

Comment:
Can we type this to anything more specific than `unknown`?

Reply:
Yup, can do `	| Record<string, string | AcquisitionABTest | AcquisitionQueryParameters>`. Patch incoming.